Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -686,18 +686,14 @@ CertReader::CertReader(std::vector<u8>&& bytes) : SignedBlobReader(std::move(byt
   if (!IsSignatureValid())
     return;
 
-  // XXX: in old GCC versions, capturing 'this' does not work for some lambdas. The workaround
-  // is to not use auto for the parameter (even though the type is obvious).
-  // This can be dropped once we require GCC 7.
-  using CertStructInfo = std::tuple<SignatureType, PublicKeyType, size_t>;
```

Comment:
I think keeping this `using CertStructInfo` is more readable.

Reply:
Fair enough, I also thought this.  Shall I remove the comment, though?

Reply:
Yeah, the comment is definitely outdated.